src/components/form-elements/draft-js-mention-selector/DraftJSMentionSelector.js

```diff
@@ -87,15 +87,47 @@ class DraftJSMentionSelector extends React.Component<Props, State> {
      * @returns {void}
      */
     componentWillReceiveProps(nextProps: Props) {
-        const { contacts } = nextProps;
+        const { editorState } = this.props;
+        const { contacts, editorState: nextEditorState } = nextProps;
+        let nextState = {};
 
         if (contacts) {
-            this.setState({ contacts }, () => {
+            nextState = { ...nextState, contacts };
```

Comment:
...nextState isn't doing anything since its defined couple of lines above.
nextState = { contacts };